tests/index.js

```diff
@@ -67,17 +73,40 @@ describe("index.js", function () {
                     const titleRegexp = new RegExp(`^# .+ \\(${fileName}\\)$`);
                     assert.ok(titleRegexp.test(lines[0]), "includes rule name in title header");
 
-                    // Second content should be fixable notice if applicable.
+                    // Decide which notices should be shown at the top of the doc.
+                    const expectedNotices = [];
+                    const unexpectedNotices = [];
+                    if (index.configs.recommended.rules[`qunit/${fileName}`]) {
+                        expectedNotices.push(CONFIG_MSG_RECOMMENDED);
+                    } else {
+                        unexpectedNotices.push(CONFIG_MSG_RECOMMENDED);
+                    }
+                    if (index.configs.two.rules[`qunit/${fileName}`]) {
+                        expectedNotices.push(CONFIG_MSG_TWO);
+                    } else {
+                        unexpectedNotices.push(CONFIG_MSG_TWO);
+                    }
                     if (index.rules[fileName].meta.fixable) {
-                        assert.equal(lines[1], "", "has blank line between title and fixable notice");
-                        assert.equal(lines[2], FIXABLE_MSG, "includes fixable notice");
-                        assert.equal(lines[3], "", "has blank line after fixable notice");
+                        expectedNotices.push(FIXABLE_MSG);
                     } else {
+                        unexpectedNotices.push(FIXABLE_MSG);
+                    }
+
+                    // Ensure that expected notices are present in the correct order.
+                    let currentLineNumber = 1;
+                    expectedNotices.forEach(expectedNotice => {
+                        assert.equal(lines[currentLineNumber], "", "has blank line before notice");
```

Comment:
Could this assert message refer to the type of notice? (See overall review comment)